src/coprocessor/dag/batch/executors/index_scan_executor.rs

```diff
@@ -34,27 +34,29 @@ impl<C: ExecSummaryCollector, S: Store> BatchIndexScanExecutor<C, S> {
         desc: bool,
         unique: bool,
     ) -> Result<Self> {
-        // Note: `unique = true` doesn't completely mean that it is a unique index scan. Instead it
-        // just means that we can use point-get for this index. In the following scenarios `unique`
-        // will be `false`:
+        // Note 1: `unique = true` doesn't completely mean that it is a unique index scan. Instead
```

Comment:
Could we move the description before this function?

Reply:
These comments are explaining the implementation, instead of documenting the function. It should not be placed as a doc comment.